.ls-lint.yml

```
@@ -0,0 +1,18 @@
+ls:
+  packages/{void-server,mock-server,core}/src:
+    .ts: kebab-case
+    .d.ts: kebab-case
+    .test.ts: kebab-case
+    .vue: PascalCase
+  packages/{void-server,mock-server,core}/hooks:
+    .ts: PascalCase
```

```suggestion
  use*.ts:
    .ts: PascalCase
```
not sure the syntax but hooks should be camel case? What do you think? Or just kebab case since its a typescript file. Not really thinking it should be pascal case.

---

Just tested it, couldn’t make it work like this!
Unfortunately, the tool seems a bit limited.